sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Release History
 
 ## 5.1.0-beta.3 (Unreleased)
+### New Features
+- Added `HealthcareOperation` long running operation for `/entities/health/jobs` endpoint released as a gated preview in v3.1-preview.3 under S tier.
```

Comment:
We don't need to tell the user what the real endpoint of the service our API is hitting. In general we expose the name of the functionality and the why, so for example something like: 
>Added the `RecognizePiiEntities` endpoint which returns entities containing Personally Identifiable Information. This feature is available in the Text Analytics service v3.1-preview.1 and above.

Reply:
same applies for analyze